lib/whois/record/parser/whois.tld.ee.rb

```diff
@@ -20,9 +18,111 @@ class Parser
       # @see Whois::Record::Parser::Example
       #   The Example parser for the list of all available methods.
       #
-      class WhoisTldEe < BaseWhoisd
-      end
+      class WhoisTldEe < Base
+        include Scanners::Scannable
+        class_attribute :status_mapping
+
+        self.status_mapping = {
+          'ok (paid and in zone)' => :registered,
+          'expired' => :expired
+        }
+
+        self.scanner = Scanners::WhoisTldEe
+
+        property_supported :available? do
```

Comment:
Please move `available?` and `registered?` after the `domain` and `domain_id`. The property definition order should be the same of https://github.com/weppos/whois/blob/master/lib/whois/record/parser/example.rb

The same applies to `disclaimer`.